middlewares/osio/osio.go

```diff
@@ -181,6 +181,16 @@ func (a *OSIOAuth) resolveByID(userID, token string, tokenType TokenType, namesp
 	return cacheData{}, err
 }
 
+func (a *OSIOAuth) resolveByIDWithoutCache(userID, token string, tokenType TokenType, namespaceName string) (cacheData, error) {
+	resolver := a.cacheResolverByID(token, tokenType, userID, namespaceName)
```

Comment:
namespaceName is always "" here. And it's also not used in cacheResolverByID()

Reply:
@alexeykazakov thanks for pointing this out, I forgot to mentioned in PR desc.

Ideally, `namespaceName` is redundant param at this point as it comes from PR (to support damenset requirement).  I have kept this param as it's to have minimal change while debugging issue with che.  So when we enable damenset support (in following PR soon), I will take care of using namespaceName meaninfully :+1: 

Hope this works for now and for you as we are still in debugging the issue and final PR has to come soon.

Reply:
OK. Approved then.